che-theia-plugins.yaml

```diff
@@ -135,20 +135,6 @@ plugins:
           value: /go:$(CHE_PROJECTS_ROOT)
     extensions:
       - https://github.com/golang/vscode-go/releases/download/v0.16.1/go-0.16.1.vsix
-  - repository:
-      url: https://github.com/haskell/vscode-haskell
-      revision: 1.1.0
-    featured: true
-    aliases:
-      - haskell/vscode-haskell
-    sidecar:
-      directory: haskell
-      name: vscode-haskell
-      memoryLimit: 3072Mi
-    extensions:
-      - https://open-vsx.org/api/haskell/haskell/1.1.0/file/haskell.haskell-1.1.0.vsix
-      - https://open-vsx.org/api/justusadam/language-haskell/3.3.0/file/justusadam.language-haskell-3.3.0.vsix
-      - https://open-vsx.org/api/gattytto/phoityne-vscode/0.0.28/file/gattytto.phoityne-vscode-0.0.28.vsix
```

Comment:
I am wondering if we could just disable `phoityne-vscode`. Not sure what is the impact.

Reply:
I didn't contribute this plugin so I don't know. Better remove it and we can add it back later.